config/ofp_flv_default.conf

```diff
@@ -0,0 +1,20 @@
+ofp_global_param: {
```

Comment:
The config files could have some comment lines about their purpose.
ofp_flv_  prefix in the file names seems unnecessary.